### PR TITLE
Add wallet authentication endpoint

### PR DESCRIPTION
**Pull request by yivlad**

### CODE REVIEW

File: packages/cli-app/README.md [hunk 1]
```diff
@@ -142,3 +142,16 @@ cli:
 The following URLs will be available:
 
 - GET `/config/config.json`
+
+###
+
+You can setup whitelist of public keys of profile identities in the Wallet Extension that can authenticated through endpoint `/wallet/auth`. In order to do so created a file in your `~/.wire` folder named `keyhole-whitelist.yml`. The file has to have the following structure:
```
Comment (rzadp) [inline packages/cli-app/README.md:148]:
```suggestion
You can setup whitelist of public keys of profile identities in the Wallet Extension that can authenticated through endpoint `/wallet/auth`. In order to do so create a file in your DXOS profile folder named `keyhole-whitelist.yml`. The file has to have the following structure:
```

File: packages/cli-app/src/server/auth.js [hunk 2]
```diff
@@ -24,6 +30,27 @@ export const authMiddleware = (loginApp) => async (req, res, next) => {
   }
 };
 
+export const walletAuthHandler = async (req, res) => {
+  const path = createPath(whitelistFile);
+  console.log(path);
```
Comment (rzadp) [inline packages/cli-app/src/server/auth.js:35]:
```suggestion
```

File: packages/cli-app/README.md [hunk 3]
```diff
@@ -142,3 +142,16 @@ cli:
...
+
+```yaml
+  publicKeys:
```
Comment (rzadp) [inline packages/cli-app/README.md:151]:
```suggestion
  whitelistedPublicKeys:
```

File: packages/cli-app/src/server/auth.js [hunk 4]
```diff
@@ -3,17 +3,23 @@
...
 import { BASE_URL } from '../config';
 import { generateQRCode, verifyToken } from '../util/OTP';
+import { createPath } from './server';
 
 const log = debug('dxos:cli-app:server:auth');
 debug.enable('dxos:*');
 
 const COOKIE_MAX_AGE = 60;
 
+const whitelistFile = '~/.wire/keyhole-whitelist.yml';
```
Comment (rzadp) [inline packages/cli-app/src/server/auth.js:18]: Looks for a constant with `~/.wire`, it's gotta be somewhere.

Comment (yivlad) [reply packages/cli-app/src/server/auth.js:18]: Doesn't look like it, almost all file paths are specified as `~/.wire/<FIlENAME>`

Comment (rzadp) [reply packages/cli-app/src/server/auth.js:18]: That's bad. Can you create an issue to change it?

**Review by rzadp — APPROVED** [review]

**Pull request merged**